### DISCUSSION TITLE
Intra-repo permalink in discussion points to commit in localhost

### DISCUSSION THREAD

**Discussion by KalleOlaviNiemitalo** [community/community#10759] [Mobile] [answered]
I posted the link <https://github.com/dotnet/docfx/blob/3acf0c1f6a4dc91363cc927bfc5fd4c61dd1c7f2/Documentation/tutorial/docfx.exe_user_manual.md#L565-L566> at <https://github.com/dotnet/docfx/discussions/7862#discussioncomment-2071839>. In that discussion, the GitHub 1.49.1 app on iOS 15.3 displays the link like this:

![image](https://user-images.githubusercontent.com/46201428/151694442-6fd7b9ea-92eb-4c01-b950-bb5078fa77c3.png)

Here, “3acf0c1” is a link to <http://localhost/dotnet/docfx/commit/3acf0c1f6a4dc91363cc927bfc5fd4c61dd1c7f2>. That doesn’t make sense as there is no HTTP server at port 80 of localhost. The link URL should instead start with `https://github.com/` and thus point to <https://github.com/dotnet/docfx/commit/3acf0c1f6a4dc91363cc927bfc5fd4c61dd1c7f2>.

The problem does not occur with the original dotnet/docfx link in this github/feedback discussion. Perhaps the problem only affects links to the same repository in which the discussion is. Here's a similar link to a file in the github/feedback repository: <https://github.com/github/feedback/blob/77a0ab5a65e84000918eeb4736b893b1a971a38b/LICENSE#L1>

**Comment by amrehman** [comment, marked as answer]
Hi! Thanks a lot for reporting the issue and providing a permalink. I can confirm I can repro the issue on my side, we will open an issue on our side and our engineers will look into this.
